QModManager/HarmonyPatches/EnableAchievements.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Original mod by AlexejheroYTB
+namespace QModManager.HarmonyPatches.EnableAchievements
+{
+    using Harmony;
+
+    [HarmonyPatch(typeof(GameAchievements), "Unlock")]
```

Comment:
Use `nameof(GameAchievements.Unlock)` instead of the raw string.

This helps us locate any breaking code changes whenever the game is updated